scripts/publish.js

```diff
@@ -75,25 +69,32 @@ async function run() {
     );
 
     // 2. Determine the appropriate npm tag to use
-    let tag = semver.prerelease(version) == null ? "latest" : "next";
+    let tag = version.includes("experimental")
+      ? "experimental"
+      : semver.prerelease(version) == null
+      ? "latest"
+      : "pre";
 
     console.log();
     console.log(`  Publishing version ${version} to npm with tag "${tag}"`);
 
     // 3. Ensure build versions match the release version
+    if (version.includes("experimental")) {
+      // FIXME: @remix-run/router is versioned differently and is only handled
+      // for experimental releases here
+      await ensureBuildVersion("router", version);
+    }
```

Comment:
In experimentals we can do this for `router` alongside since they all share a version